docs/reference/connection/ssh.md

```diff
@@ -0,0 +1,88 @@
+---
+description: Defines the whole behavior of the SSH protocol by itself for a user who is connecting to Bifröst.
+---
+# SSH connection
+
+Defines the whole behavior of the SSH protocol by itself for a user who is connecting to Bifröst.
```

Comment:
Defines the behavior of the SSH protocol for a user who is connecting to Bifröst.